recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+{% set file_url = "http://gforge.inria.fr/frs/download.php/file/34618/scotch_6.0.4.tar.gz" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: scotch_{{ version }}.tar.gz
+  md5: {{ md5 }}
+  url: {{ file_url }}
+  patches:
+    - clock_gettime.patch  # [osx]
```

Comment:
To be honest I don't know. It was done in another recipe I found and on the homebrew version.